DEVELOPMENT.md

```diff
@@ -0,0 +1,75 @@
+# module name 
```

Comment:
We have a similar file in .github/CONTRIBUTING.md. Is this file still relevant? Maybe we should merge it?